lib/passageidentity/user_api.rb

```diff
@@ -240,23 +185,76 @@ def signout(user_id:)
 
     private
 
+    def create_v2(args: {})
+      warn '[DEPRECATED] the `create` method parameters will change to `args: CreateUserArgs`.' \
+      'Parameters will change on or after 2025-1.'
+
+      begin
+        response = @user_client.create_user(@app_id, args, @req_opts)
+        response.user
+      rescue Faraday::Error => e
+        raise PassageError.new(
+          status_code: e.response[:status],
+          body: e.response[:body]
+        )
+      end
+    end
+
+    def update_v2(user_id:, options: {})
+      warn '[DEPRECATED] the `update` method parameters will change to `userId: string, ' \
+           'options: UpdateUserArgs`. Parameters will change on or after 2025-1.'
+
+      user_exists?(user_id)
```

Comment:
this check is already called in the original update method and can be removed here